src/systems/selection/draw_multiple_selection_box.rs

```diff
@@ -1 +1,74 @@
-// TODO add system
+use bevy::{
+    math::{Vec2, Vec3},
+    prelude::{Commands, Entity, EventReader, EventWriter, Query, Transform},
+};
+
+use crate::{
+    components::user_interface::multiple_selection_box::MultipleSelectionBox,
+    events::{
+        input_events::MouseLeftClickEvent,
+        spawn_sprite_event::{SpawnSprite, SpawnSpriteEvent},
+    },
+};
+
+// TODO create queries
+pub fn draw_multiple_selection_box(
+    mut commands: Commands,
+    mut select_event_reader: EventReader<MouseLeftClickEvent>,
+    mut selection_query: Query<(Entity, &mut MultipleSelectionBox)>,
+    mut spawn_sprite_writer: EventWriter<SpawnSpriteEvent>,
+) {
+    let Some(select) = select_event_reader.read().last() else {
+        return;
+    };
+
+    let cursor_position = select.cursor_world_position;
+
+    match !selection_query.is_empty() {
+        true => match !select.just_released {
+            true => {
+                // Resize Selection Box
+                let selection_box = selection_query.single_mut();
+                let mut multiple_selection_box = selection_box.1;
+                multiple_selection_box.end = cursor_position;
+
+                let midpoint = multiple_selection_box
+                    .start
+                    .lerp(multiple_selection_box.end, 0.5)
+                    .extend(50.0);
+                let size_x = multiple_selection_box.end.x - multiple_selection_box.start.x;
+                let size_y = multiple_selection_box.end.y - multiple_selection_box.start.y;
+
+                commands
+                    .entity(selection_box.0)
+                    .insert(*multiple_selection_box)
+                    .insert(
+                        Transform::from_translation(midpoint)
+                            .with_scale(Vec3::new(size_x, size_y, 6.0)),
+                    );
+            }
+            false => {
+                // TODO trying to despawn in select_multiple_sprites
```

Comment:
Shouldn't need this match statement as it should be handled in select_multiple_sprites